geomesa-kafka/geomesa-kafka-datastore/pom.xml

```diff
@@ -70,6 +79,13 @@
             <artifactId>kafka_2.10</artifactId>
         </dependency>
         <dependency>
+            <!--This is a compile dependency of Kafka and also KafkaGeoMessageConsoleConsumer -->
+            <groupId>net.sf.jopt-simple</groupId>
+            <artifactId>jopt-simple</artifactId>
+            <version>3.2</version>
+            <scope>compile</scope>
```

Comment:
I've moved the version management to the parent where there is already some Kafka related dependency management.  Going forwared should all dependency management be added / moved to the root?

Additionally, I noticed that geomesa-kafka was using a different version of Guava than that managed by the root.  I removed the override.